openlibrary/templates/account/borrow.html

```diff
@@ -169,7 +169,7 @@ <h1>$_("Books You're Waiting For")</h1>
         <thead>
             <tr>
                 <th class="titles" colspan="2">
-                $ungettext("%d book", "%d books", len(waitinglist), len(waitinglist))
+                $ungettext("1 book", "%(count)d books", len(waitinglist), count=len(waitinglist))
```

Comment:
Note for future reference, if we probably don't need to name the number if there's only one and it's `count`; I think that can be deduced, and they look easier to read without.